docs/start/quickstart_group.md

```diff
@@ -453,6 +453,23 @@ For the [DKG](../charon/dkg.md) to complete, all operators need to be running th
               <img src="/img/DKGExample.png" alt="Connect to peers in logs tab" />
             </li>
             </ol>
+     <h2>Create a DV Node Backup</h2>
+     It is important to back up all artefacts generated by the DKG ceremony, and your node ENR private key. The below steps will show you how to download your keys & node artefacts.
```

Comment:
We use a mix of artifact and artefact, but your way is the british english way, so maybe we'll edit the rest to this way for consistency. No action needed.